src/nspawn/nspawn.c

```diff
@@ -4085,6 +4077,7 @@ int main(int argc, char *argv[]) {
 
         log_parse_environment();
         log_open();
+        cg_unified_flush();
```

Comment:
I don't think this is needed at all, `CGROUP_UNIFIED_UNKNOWN` is the default value assigned at program startup.